gradle/versions.gradle

```diff
@@ -50,7 +50,7 @@ dependencyManagement {
 
     dependency 'io.pkts:pkts-core:3.0.3'
 
-    dependency 'org.mock-server:mockserver-junit-jupiter:5.11.0'
+    dependency 'org.mock-server:mockserver-junit-jupiter:5.10.0'
```

Comment:
v5.11 of mockserver seems to cause some transitive dependency issues relating to jackson. I'll investigate what specifically is causing the issue and address it differently